src/django/api/models.py

```diff
@@ -406,142 +408,241 @@ class FacilityClaim(models.Model):
     facility_average_lead_time = models.CharField(
         max_length=200,
         null=True,
         blank=True,
+        verbose_name='Average lead time',
         help_text='The editable facilty avg lead time for this claim.')
     point_of_contact_person_name = models.CharField(
         max_length=200,
         null=True,
         blank=True,
+        verbose_name='Contact person',
         help_text='The editable point of contact person name')
     point_of_contact_email = models.EmailField(
         null=True,
         blank=True,
+        verbose_name='Contact email',
         help_text='The editable point of contact email')
     point_of_contact_publicly_visible = models.BooleanField(
         null=False,
         default=False,
+        verbose_name='Is contact visible',
         help_text='Is the point of contact info publicly visible?')
     office_official_name = models.CharField(
         max_length=200,
         null=True,
         blank=True,
+        verbose_name='Office official name',
         help_text='The editable office name for this claim.')
     office_address = models.CharField(
         max_length=200,
         null=True,
         blank=True,
+        verbose_name='Office address',
         help_text='The editable office address for this claim.')
     office_country_code = models.CharField(
         max_length=2,
         null=True,
         blank=True,
         choices=COUNTRY_CHOICES,
+        verbose_name='Office country code',
         help_text='The editable office country code for this claim.')
     office_phone_number = models.CharField(
         max_length=200,
         null=True,
         blank=True,
+        verbose_name='Office phone number',
         help_text='The editable office phone number for this claim.')
     office_info_publicly_visible = models.BooleanField(
         null=False,
         default=False,
+        verbose_name='Is office publicly visible',
         help_text='Is the office info publicly visible?')
     parent_company = models.ForeignKey(
         'Contributor',
         related_name='parent_company',
         null=True,
         default=None,
         on_delete=models.PROTECT,
+        verbose_name='Parent company',
         help_text='The parent company of this facility claim.')
 
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     history = HistoricalRecords()
 
+    default_change_includes = (
+        'facility_name',
+        'facility_address',
+        'facility_phone_number',
+        'facility_website',
+        'facility_description',
+        'facility_minimum_order_quantity',
+        'facility_average_lead_time',
+        'point_of_contact_person_name',
+        'point_of_contact_email',
+        'office_official_name',
+        'office_address',
+        'office_country_code',
+        'office_phone_number',
+        'parent_company',
+    )
```

Comment:
#650 will have a few new fields to add to this list, but it should be easy to adjust.